### PR TITLE
Update README with authentication clarification

### PR DESCRIPTION
**Pull request by accraze**
Added clarification that the password is the HTTP password generated in the user account settings.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,6 +88,7 @@ machine gerrithostname.org
     login my-gerrit-username
     password xxxx
 ```
+Note: Gerrit's HTTP basic authentication expects the HTTP password from the user’s account settings page. Make sure to generate one, copy it and add it as the password in your auth-source.
```
Comment (thisch) [inline README.md:91]: This depends a bit on the configuration of gerrit. At work we're using `auth_type="LDAP"` and `git_basic_auth_policy="LDAP"`, for which no separate gerrit http auth password needs to be generated.

I would mention here that the way the user is authenticated depends on the gerrit configuration. If there is a 

image

section in the user's account settings page, then a password needs to be generated.

**Review by thisch — COMMENTED** [review]
Thx a lot for your contribution!

File: README.md (hunk 1 above)
Comment (accraze) [reply README.md:91]: Ah yeah this is a great point. I am using Gerrit 3.2.7 with `auth_type="LDAP"` and `git_basic_auth_policy="HTTP_LDAP"`, which would explain the difference. I will update the patch shortly.

File: README.md [hunk 2]
```diff
@@ -88,6 +88,7 @@ machine gerrithostname.org
     login my-gerrit-username
     password xxxx
 ```
+Note: Depending on your auth configuration, Gerrit may expect a generated HTTP password (ex. if you have `git_basic_auth_policy="HTTP_LDAP"`). If there is an HTTP credentials section in your user's account settings page, then an HTTP password needs to be generated and supplied in your auth-source file.
```
Comment (thisch) [inline README.md:91]: Do you use git_basic_auth_policy="HTTP_LDAP" on your gerrit server and see a HTTP credentials section in the users' account settings page?  If yes, LGTM!

Comment (accraze) [reply README.md:91]: yep! just checked my server-info and confirmed HTTP_LDAP

**Comment by thisch** [comment]
Thx again for your contribution.

**Pull request merged**